client/src/redux/actions/initialSearch.js

```diff
@@ -62,34 +62,35 @@ export function searchAddressFlow(address, latLngHelper) {
   };
 }
 
-export function setCampaignInformation(campaignInfo) {
-  return {
-    type: 'SET_CAMPAIGN_INFORMATION',
-    promise: createApiRequest('api/campaigns', 'POST', { ...campaignInfo })
-  };
-}
-
 export function selectAddress(value) {
-  return {
-    type: 'SELECT_ADDRESS',
-    value
-  };
+	return {
+		type: 'SELECT_ADDRESS',
+		value
+	};
 }
 
-export function createCampaignFailure(error) {
-  return {
-    type: 'CREATE_CAMPAIGN_FAILURE',
-    error
-  };
-}
+// export function setCampaignInformation(campaignInfo) {
+//   return {
+//     type: 'SET_CAMPAIGN_INFORMATION',
+//     promise: createApiRequest('api/campaigns', 'POST', { ...campaignInfo })
+//   };
+// }
 
-export function createCampaign(campaignInfo) {
-  return async (dispatch) => {
-    const { response } = await dispatch(setCampaignInformation(campaignInfo));
-    if (response.errors) {
-      dispatch(createCampaignFailure(response.errors));
-    } else {
-      browserHistory.push(`/campaign/${response.data._id}`);
-    }
-  };
-}
+
+// export function createCampaignFailure(error) {
+//   return {
+//     type: 'CREATE_CAMPAIGN_FAILURE',
+//     error
+//   };
+// }
+
```

Comment:
are you leaving these commented code blocks in for any particular purpose? If not, please remove them. Thanks!